src/components/Nav/DashboardNav.jsx

```diff
@@ -85,7 +85,13 @@ function DashboardNav(props) {
           ))}
 
           <DropdownItem divider />
-          <DropdownItem onClick={() => localStorage.removeItem("token")}>
+          <DropdownItem
+            onClick={() => {
+              localStorage.removeItem("token");
```

Comment:
I understand what you removed, but not why the second or third line are needed. Just above my pay-grade so far.

Reply:
I didn't change anything here, prettier did that.